apps/api/src/__tests__/e2e/data-attributes.test.ts

```diff
@@ -0,0 +1,156 @@
+import { describe, it, expect } from '@jest/globals';
```

Comment:
Wrong place to put the test, integrate it into `snips/v2/scrape.test.ts` using the existing helpers there